src/appleseed/renderer/modeling/camera/thinlenscamera.cpp

```diff
@@ -883,6 +794,22 @@ DictionaryArray ThinLensCameraFactory::get_input_metadata() const
             .insert("use", "optional")
             .insert("default", "-0.001"));
 
+    metadata.push_back(
+        Dictionary()
+            .insert("name", "shift_x")
+            .insert("label", "Shift X")
+            .insert("type", "text")
```

Comment:
Should be `numeric` with proper min/max bounds (probably soft ones); `text` is really meant for strings.